teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services
```

Comment:
![cus](https://github.com/jecode93/oop-school-library/assets/89055502/20913619-22e0-43d7-9c35-91a0cf6f37b5)
- [ ] As shown in the project requirement above, you are using the wrong name for your method, it should be `can_use_services?` not `can_use_services`, please kindly correct this and resubmit for approval :+1: